modules/main/package.json

```diff
@@ -49,5 +53,15 @@
     "@deck.gl/react": "9.0.0-beta.10",
     "@deck.gl/widgets": "9.0.0-beta.10"
   },
+  "peerDependencies": {
+    "@arcgis/core": "^4.0.0",
+    "react": ">=16.3.0",
+    "react-dom": ">=16.3.0"
+  },
+  "peerDependenciesMeta": {
+    "@arcgis/core": {"optional": true},
```

Comment:
I don't know `peerDependenciesMeta` existed — that's so helpful! 🤯

Reply:
`optionalDependencies` still exist. `peerDependenciesMeta` is for peer dependencies instead of dependencies.

react-map-gl has been [using it](https://github.com/visgl/react-map-gl/blob/d10dc62c601166ac6a7b458a44521759b89faccf/package.json#L62) and nobody has complained.